### DISCUSSION TITLE
android app

### DISCUSSION THREAD

**Discussion by fu8765** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#171] [APK sharing]
Is it possible to start releasing android versions?
Adb permissions are easy with developer mode and Shizuku, no need for root to disable packages

**Comment by AnonymousWP** [comment]
A no for me, for a variety of reasons:

- Costs a lot of time to maintain (UAD-ng is already quite a lot of work to maintain), apart from our busy lives;
- When you brick your phone using UAD-ng, you can still connect it to your PC and fix it. If navigation buttons or gestures or other critical functionality isn't working anymore, you won't even be able to use UAD as an app, because you'll have to navigate to it.

**Comment by shuvashish76** [comment]
There are couple of options :
- [AppManager](https://f-droid.org/en/packages/io.github.muntashirakon.AppManager/) - Uses its own [fork](https://github.com/MuntashirAkon/android-debloat-list) (WIP) of UAD
- [De-Bloater](https://f-droid.org/en/packages/com.sunilpaulmathew.debloater/) - Uses UAD
- [Canta](https://f-droid.org/en/packages/org.samo_lego.canta/) - Uses UAD-ng

@fu8765 For your use-case you can try Canta since it supports Shizuku & with [v1.2.1](https://github.com/samolego/Canta/releases/tag/1.2.1) now it uses UAD-ng. 🎉

**Reply by shuvashish76** [reply]
@AnonymousWP Request to add De-Bloater to [Friends list](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation#friends). I just noticed in v[0.32](https://github.com/sunilpaulmathew/De-Bloater/releases/tag/v0.32) they're now using UAD-ng :)

**Reply by AnonymousWP** [reply]
Thanks, will create a new issue for it.
